packages/tooling/cfp-template/template/fastconfig.json

```diff
@@ -0,0 +1,28 @@
+{
+    "packageJson": {
```

Comment:
Is the purpose of this file to track the state of the package.json file at the time it was generated?

Reply:
Yes, originally I had this as a package.json file within the template folder, but yarn had an issue with this in a monorepository setting and I saw where it got confusing. I have plans to add a check to make sure we have consistency between the package.json and this area of the config.